.gitignore

```diff
@@ -150,6 +150,8 @@ dataflow/tests/issue3447/Out.txt
 dataflow/tests/issue3447/*.class
 dataflow/tests/live-variable/Out.txt
 dataflow/tests/live-variable/*.class
+dataflow/tests/reachingdef/Out.txt
```

Comment:
@AndrewShf In a separate PR, can you fix the directory names above from `busy-expression` to `busyexpr`? I think that's the wrong folder, isn't it?

Reply:
Sorry, I forgot to rename it in .gitignore. I just made a PR to fix this

Reply:
Thanks for #229!